timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -237,6 +237,24 @@ public void leaderIdChangesAcrossFailovers() {
         }
     }
 
+    // @Test
+    // public void leaderIdChangesAcrossFailoversStressTest() {
+    //     Set<LeaderTime> leaderTimes = new HashSet<>();
+    //     leaderTimes.add(client.namespacedConjureTimelockService().leaderTime());
+    //
+    //     int leaderElectionCount = 11;
+    //     for (int i = 0; i < leaderElectionCount; i++) {
+    //         cluster.failoverToNewLeader(client.namespace());
```

Comment:
This test times-out, we do not need to have this stress test really as the leader election rate health check will fire in non-test environment.